installation-manager/src/main/java/org/wildfly/core/instmgr/cli/UpdateCommand.java

```diff
@@ -38,8 +39,10 @@ public class UpdateCommand extends AbstractInstMgrCommand {
     private List<String> repositories;
     @Option(name = "local-cache")
     private File localCache;
-    @Option(name = "no-resolve-local-cache", hasValue = false)
+    @Option(name = NO_RESOLVE_LOCAL_CACHE_OPTION, hasValue = false, activator = AbstractInstMgrCommand.NoResolveLocalCacheActivator.class, defaultValue = "true")
```

Comment:
A minor comment, this option ha no value, setting the default value to true is not needed.